components/list/list-item-drag-handle.js

```diff
@@ -0,0 +1,196 @@
+
+import '../button/button-icon.js';
+import '../icons/icon.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { buttonStyles } from '../button/button-styles.js';
+import { getFirstFocusableDescendant } from '../../helpers/focus.js';
+
+const keyCodes = Object.freeze({
+	END: 35,
+	HOME: 36,
+	UP: 38,
+	DOWN: 40,
+	SPACE: 32,
+	ENTER: 13,
+	ESC: 27,
+	TAB: 9,
+	LEFT: 37,
+	RIGHT: 39,
+
+});
+
+export const dragActions = Object.freeze({
+	first: 'last',
+	end: 'first',
+	up: 'up',
+	down: 'down',
+	active: 'keyboard-active',
+	save: 'keyboard-deactivate-save',
+	cancel: 'keyboard-deactivate-cancel',
+	nextElement: 'next-element',
+	previousElement: 'previous-element'
+});
+
+class listItemDragHandle extends LitElement {
+
+	static get properties() {
+		return {
+			keyboardActive: { type: Boolean, attribute: 'keyboard-active', reflect: true },
```

Comment:
Looks like this should be private...?

Reply:
I have a feeling this might need to be public. I have a method that will flip this into keyboard but honestly, I think there should be a better solution for this workflow and so I am going to make this private so when/if it comes up to be public we can see about another solution to the problem it is solving. 

The reason it is public right now is I copied my prototyped version of this component and there it required to be public. So similar the the `sub-icon` class this might be a relic of the prototype that doesn't need to be in.